src/cmds/inpost.ts

```diff
@@ -0,0 +1,59 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { Message } from 'discord.js';
+import { fetch } from 'undici';
+export async function run(
+    client: FluorineClient,
+    message: Message,
+    args: string[]
+) {
+    if (!args[0]) {
+        return message.reply(
+            client.language.get(message.guild.preferredLocale, 'INPOST_ARGS')
+        );
+    }
+    let statusURL;
+    if (message.guild.preferredLocale === 'pl') {
+        statusURL = 'https://api-shipx-pl.easypack24.net/v1/statuses';
+    } else {
+        statusURL = 'https://api-shipx-pl.easypack24.net/v1/statuses?lang=en';
+    }
+    const statuses: any = await (await fetch(statusURL)).json();
```

Comment:
dont use double await
```suggestion
    const statuses: any = await fetch(statusURL).then(res => res.json());
```